portfolio/src/main/webapp/index.html

```diff
@@ -2,14 +2,26 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Karishma</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Karishma</h1>
+      <p>This is Karishma.</p>
+      <img src="/images/IMG-1.jpg/" alt="IMG-1" height="300" width="420">
```

Comment:
Most probably I am missing something here, but -

a) Running the project with img src = "/images/IMG-1.jpg/", gives me this - 
![Screenshot 2020-07-12 at 6 06 00 PM](https://user-images.githubusercontent.com/14831291/87246387-baeea180-c46a-11ea-83ec-140fb8d748ba.png)
b) Running the project with img src = "images/IMG-1.jpg", gives me this - 
![Screenshot 2020-07-12 at 6 05 28 PM](https://user-images.githubusercontent.com/14831291/87246400-d9549d00-c46a-11ea-9be7-572351e77ddb.png)

I ran this locally without a server (just downloaded your whole repository and merged this PR in it). Do you think we should follow - https://teamtreehouse.com/community/difference-between-links-with-forward-slashes-and-relative-links and remove the extra backslashes? What did I miss here?